scripts/Interface/ui/sans_isis/sans_data_processor_gui.py

```diff
@@ -539,6 +540,9 @@ def _on_save_other_button_pressed(self):
     def _on_help_button_clicked(self):
         if PYQT4:
             proxies.showCustomInterfaceHelp('ISIS SANS v2')
+        else:
+            InterfaceManager().showHelpPage('qthelp://org.sphinx.mantidproject.4.0/doc/'
```

Comment:
Is the `4.0` in the URL a Mantid version number? If so is it possible to remove this so it always shows the version of Mantid that is running?

Reply:
It is; I've removed it now